packages/fxa-payments-server/public/index.html

```diff
@@ -19,23 +19,57 @@
     <meta name="fxa-feature-flags" content="__FEATURE_FLAGS__" />
 
     <script src="https://js.stripe.com/v3/"></script>
+    <style>
+      html {
+        background-color: hsl(240, 60%, 99%);
+      }
+
+      body {
+        align-items: center;
+        display: flex;
+        flex-direction: column;
+        justify-content: center;
+        min-height: 100vh;
+        padding: 48px 0 0;
+      }
+
+      #loading-spinner img {
+        animation: .9s spin infinite linear;
+        display: block;
+        height: 36px;
+        overflow: hidden;
+        position: absolute;
+        text-indent: 100%;
+        top: calc(50% - 36px);
+        white-space: nowrap;
+        width: 36px;
+      }
+
+      @keyframes spin {
+        from {
+          transform: rotate(0deg);
+        }
+
+        to {
+          transform: rotate(360deg);
+        }
+      }
+    </style>
   </head>
 
   <body>
-    <div id="root">
```

Comment:
`id="root"` is where the React app renders, once it's loaded. The contents here are just kind of pre-loading stuff until React comes in and replaces it all. 

Moving the loading spinner above this in the DOM means the app is never visible. There's also a bunch of repeated elements here (static-footer, stage, etc) because React renders them in root.

<img width="280" alt="Capture" src="https://user-images.githubusercontent.com/21687/62958203-11f77c80-bdab-11e9-8a33-92c98eed79e7.PNG">